calculator/src/App.js

```diff
@@ -0,0 +1,73 @@
+import { Component } from "react";
+import "./App.css";
+import Buttons from "./Components/Buttons";
+import Input from "./Components/Input";
+
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { input: "0", operator: "", previousNumber: ""};
+  }
+
+
+ 
+
+  handleInput = (e) => {
+    
+    this.setState({ input: e.target.value});
+  };
```

Comment:
Make sure to use a code formatter like prettier (option + shift + f) is the shortcut